lib/modules/manager/custom/regex/readme.md

```diff
@@ -1,4 +1,4 @@
-With the `regex` manager you can configure Renovate so it finds dependencies that are not detected by its other built-in package managers.
+With `customManagers` using `regex` you can configure Renovate so it finds dependencies that are not detected by its other built-in package managers.
```

Comment:
```suggestion
Renovate's built-in package managers may not find all your dependencies.
But you can tell Renovate how to find a missing dependency.
In short: write your own regex, and put the regex in the `customManagers` field.
Read the rest of the page to learn more.
```

How about something like this? Check if this is technically correct before applying. :smile:

Reply:
🤷‍♂️

Reply:
Since we are already in the regex manager's readme its fine to call it rege manager directly, I think.